### PR TITLE
Switch from install to clean install

### PR DESCRIPTION
**Pull request by alexanderkasten**
## Changes

 1. Switch from normal install to clean install
 2. Remove package-lock from gitignore
 3. Add package-lock

## Issues

PR: #50 

## How to test the changes

> Describe how others can test your changes (not required when fixing typos, linter errors and such)

### CODE REVIEW

File: typescript/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "@types/socket.io": "^2.1.0",
     "@types/socket.io-client": "^1.4.32",
     "eslint": "^5.16.0",
+    "eslint-plugin-import": "^2.17.3",
```
Comment (ElRaptorus) [inline typescript/package.json]: What do you need this for?

Comment (alexanderkasten) [reply typescript/package.json]: npm lint

Comment (alexanderkasten) [reply typescript/package.json]: https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fconsumer_api_client/detail/feature%2Fswitch_from_install_to_clean-install/2/pipeline

Comment (ElRaptorus) [reply typescript/package.json]: Strange. This package is actually included with [eslint-config](https://github.com/essential-projects/eslint-config/blob/develop/package.json).
It should not be necessary to reference it here.

Are you experiencing this anywhere else as well?

Comment (ElRaptorus) [reply typescript/package.json]: I cannot reproduce this locally. I'd rather not add more dependencies here than are necessary. I'll have a closer look at this.

Comment (alexanderkasten) [reply typescript/package.json]: > Strange. This package is actually included with [eslint-config](https://github.com/essential-projects/eslint-config/blob/develop/package.json).
> It should not be necessary to reference it here.
> 
> Are you experiencing this anywhere else as well?

No, only this package.

I have tested it locally and I was able to reproduce the jenkins error

Comment (ElRaptorus) [reply typescript/package.json]: I think your package lock was corrupted, somehow. 

I removed the dependency, did a full reinstall and now it works:
https://ci.process-engine.io/blue/organizations/jenkins/process-engine_node-lts%2Fconsumer_api_client/detail/feature%2Fswitch_from_install_to_clean-install/4/pipeline

**Pull request merged**